pkg/maps/metricsmap/metricsmap.go

```diff
@@ -187,4 +274,14 @@ func init() {
 		MaxEntries: MaxEntries,
 		Pinning:    ebpf.PinByName,
 	})
+
+	// Register metrics map collector only once
+	once.Do(func() {
+		c := newMetricsMapCollector()
```

Comment:
nit: no need for this tmp var. Call directly? `metrics.Register(newMetricsCollector())`